updater/scripts/git-download.sh

```diff
@@ -20,15 +20,20 @@ if ghe_greater_equal "2.11.0" ; then
     # most recent log files (because the information from yesterday may or not
     # be rotated already).
     CAT_LOG_FILE="zcat -f /var/log/github-audit.{log.1*,log} | grep -F '$(date --date='yesterday' +'%b %_d')'"
+
+    # The order in github-audit.log in GHE 2.12.x has been changed, to pick the right order PERL_REGEX is created.
+    PERL_REGEX='print if s/.*"cloning":([^,]+).*"program":"upload-pack".*"repo_name":"([^"]+).*"uploaded_bytes":([^,]+).*"user_login":"([^"]+).*/\2\t\4\t\1\t\3/'
```

Comment:
Wouldn't that use the new regex for 2.11, too? The function above checks for `greater_equal`!

Reply:
You are right, then I recommend separating the version check for perl regex and the log file into 2 different ifs.